src/python/pants/backend/scala/compile/scalac_test.py

```diff
@@ -384,7 +384,8 @@ def test_compile_with_scalac_plugin(rule_runner: RuleRunner) -> None:
 
                 scalac_plugin(
                     name = "acyclic",
-                    artifact = ":acyclic_lib",
+                    # TODO: Support relative addresses.
```

Comment:
This was always broken, but now we're more eagerly validating the address so it makes a difference. I can try fixing in a followup? We need to use `AddressInput.parse(relative_to)`